### PR TITLE
[Merged by Bors] - malfeasance proof: add received time

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation
part of #4089

## Changes
add received time for malfeasance proof. preparing implementation for https://community.spacemesh.io/t/grading-atxs-for-the-active-set/335

### CODE REVIEW

File: sql/migrations/0001_initial.sql [hunk 1]
```diff
@@ -25,15 +25,17 @@ CREATE INDEX ballots_by_layer_by_pubkey ON ballots (layer asc, pubkey);
 
 CREATE TABLE identities
 (
-    pubkey VARCHAR PRIMARY KEY,
-    proof  BLOB
+    pubkey   VARCHAR PRIMARY KEY,
+    proof    BLOB,
+    received INT NOT NULL
```
Comment (dshulyak) [inline sql/migrations/0001_initial.sql]: we can't extend 0001_initial anymore. maybe we can merge them during checkpoint recovery, but i am not sure

**Review by fasmat — APPROVED** [review]
Besides the already mentioned need for a DB migration by @dshulyak this PR LGTM

**Comment by countvonzero** [comment]
bors try

**Comment by countvonzero** [comment]
@dshulyak can you sign off on this change?

one thing i am considering is to rename the new file to 0002_next.sql and include all schema changes until the next release. 
when 0003_next.sql is created, rename 0002_next.sql to 0002_v1.1.sql or something like that.

**Review by dshulyak — APPROVED** [review]
makes sense to keep unreleased changes in same file, as for naming i don't know whats best, maybe just number

**Comment by countvonzero** [comment]
> makes sense to keep unreleased changes in same file, as for naming i don't know whats best, maybe just number

thanks. 

currently the parsing requires $version_$name.sql. 
name is always for readability anyway. i think release version is good way to communicate.

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**